code/game/machinery/cryopod.dm

```diff
@@ -324,6 +324,20 @@
 			else
 				W.loc = loc
 
+	// Skip past any cult sacrifice objective using this person
+	if(GAMEMODE_IS_CULT && is_sacrifice_target(occupant.mind))
+		var/datum/game_mode/cult/cult_mode = ticker.mode
+		var/list/p_s_t = cult_mode.get_possible_sac_targets()
+		if(p_s_t.len)
+			cult_mode.sacrifice_target = pick(p_s_t)
+			for(var/datum/mind/H in ticker.mode.cult)
+				if(H.current)
+					to_chat(H.current, "<span class='danger'>[ticker.mode.cultdat.entity_name]</span> murmurs, <span class='cultlarge'>[occupant] is beyond your reach. Sacrifice [cult_mode.sacrifice_target.current] instead...</span></span>")
+			for(var/mob/dead/observer/O in player_list)
```

Comment:
I don't think it's a good idea to tell ghosts this--you're going to have a lot of players who have nothing to do with cult or who are being revived from cult knowing, suddenly knowing how the sacrifice target is.

This *will* be used to metagame.